src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -12,8 +12,20 @@ const createZomeCall = instanceCreateZomeCall(INSTANCE_ID)
 
 const MOCK_DEADLINE = '4019-01-02T03:04:05.678901234+00:00'
 
+/* Creates an array of all transactions that inlcude new counterparties for a provided transaction list and counterparty list */
+export const findnewCounterpartiesFromList = (transactionList = [], counterpartyList = []) => {
+  // eslint-disable-next-line array-callback-return
+  const newCouterpartyList = transactionList.filter(({ counterparty }) => {
```

Comment:
typo in this name, and I would recommend using the array naming convention above. so `newCounterparties`

Reply:
although it's not actually an array of counterparties, it's an array of transactions, so maybe `newCounterpartyTransactions`